### PR TITLE
Bug/fixing build errors from practice hooks

### PR DESCRIPTION
**Pull request by brandonfryer**

### CODE REVIEW

File: components/Gallery.js [hunk 1]
```diff
@@ -5,34 +5,34 @@ const Gallery = () => {
         <>
             <div className={styles.row}> 
                 <div className={styles.column}>
-                    <img src="/fam/04.jpg" />
-                    <img src="/fam/01.jpg" />
-                    <img src="/fam/17.png" />
-                    <img src="/fam/06.png" />
-                    <img src="/fam/07.jpg" />
+                    <img src="/fam/04.jpg" alt="" />
```
Comment (brianfryer) [inline components/Gallery.js]: This is great for resolving the `warning`, but let's find some time to put some _real_ content here.

If you're unfamiliar with how to format good `alt` attributes, look no further: https://moz.com/learn/seo/alt-text

File: pages/contact.js [hunk 2]
```diff
@@ -10,15 +10,15 @@ export default function Contact() {
                     name="name"
                     className={styles.name}
                     id="inputName"
-                    oninput="checkInput()"
+                    onInput="checkInput()"
```
Comment (brianfryer) [inline pages/contact.js]: The value of this `onInput` attribute is _calling_ `checkInput`. Instead, it should simply reference the function name without the parens. For example:

```
onInput="checkInput"
```

Comment (brianfryer) [reply pages/contact.js]: Also, instead of rolling your own validator, use https://react-hook-form.com/

**Review by brianfryer — CHANGES_REQUESTED** [review]

File: components/Gallery.js (hunk 1 above)
Comment (brandonfryer) [reply components/Gallery.js]: check

File: pages/contact.js (hunk 2 above)
Comment (brandonfryer) [reply pages/contact.js]: thanks for the demonstration! check

**Pull request merged**